src/app/admin/ownership/ownership.component.html

```diff
@@ -126,11 +125,11 @@ <h3>Coming Soon</h3>
                 <div class="ml-2">
                     <span class="m-2 mr-3 p14">
                         <input type="checkbox" id="vehicle1" name="vehicle1" value="Bike" unchecked />
-                        <label for="vehicle1" class="mr-2 p14"> &nbsp; Mobile Number</label>
+                        <label for="vehicle1" class="mr-2 p14"> &nbsp; {{'MOBILE_NUMBER' | translate}}</label>
     
                         <select name="visibility" id="visibility" class="p14 w-185 p-2 fieldInput">
                             <option value="public"> <i class="fa fa-ellipsis-h" aria-hidden="true"></i>
-                                educationboard.email
+                                {{'EDUCATIONBOARD_EMAIL' | translate}}
```

Comment:
@suvarnakale here we are showing value or its just label? please confirm once `educationboard.email`